docs/installation.rst

```diff
@@ -15,33 +15,86 @@ to avoid version conflicts with other packages you may have installed.
 User Installation
 -----------------
 
-Installing the released version can be done using pip::
+Some of Jdaviz's dependencies require non-Python packages to work
+(particularly the front-end stack that is part of the Jupyter ecosystem).
+We recommend using `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+to easily manage a compatible Python environment for Jdaviz; it should work
+with most modern shells, except CSH/TCSH.
 
-   pip install jdaviz --upgrade
+Once it is installed, we recommend you create a new environment rather than
+installing everything into the ``base`` environment, for example::
+
+    conda create -n jdaviz-env python=3.9
+    conda activate jdaviz-env
+
+For those using ``conda``, we recommend pulling the following from ``conda``
+instead of ``pip``. You could always try ``pip`` but sometimes the installation
+might not work properly for some people::
+
+    conda install bottleneck
+    conda install -c conda-forge notebook
+    conda install -c conda-forge jupyterlab
+    conda install -c conda-forge voila
+
+You might also want to enable ``ipywidgets`` notebook extension, as follows::
+
+    jupyter nbextension enable --py widgetsnbextension
```

Comment:
It's been a while since I had to install anything with conda, now that I think about it. I would advocate for moving this section to below the simplest version (i.e. just the pip install) so that people aren't intimidated/try the simple pip install first. Could then change the text to

```
Users occasionally encounter problems running the pure ``pip`` install above. For those 
using ``conda``, some problems may be resolved by pulling the following from ``conda``
instead of ``pip``::
```

Reply:
Okay, I think I changed it the way you wanted. Please re-review. Thanks!